src/effects/nyquist/Nyquist.cpp

```diff
@@ -1530,8 +1530,10 @@ bool NyquistEffect::ProcessOne()
       unsigned int l;
       auto ltrack = * mOutputTracks->Any< LabelTrack >().begin();
       if (!ltrack) {
+         auto newTrack = std::make_shared<LabelTrack>();
+         newTrack->SetName(inputTracks()->MakeUniqueTrackName(LabelTrack::GetDefaultName()));
```

Comment:
The repetition of making a track, then setting its default name with a unique number, is repeated in a few places.  I would suggest a static member function of LabelTrack to simplify this repetition.

Reply:
Makes sense, thanks